### ISSUE TITLE
[FEATURE] Add session get status function to the library

### ISSUE THREAD

**Issue by majkiee** [Adyen/adyen-node-api-library#1214]
**Is your feature request related to a problem? Please describe.**
The get session status endpoint is missing from the node api

**Describe the solution you'd like**
Please add support for the session status endpoint: 
[Documentation](https://docs.adyen.com/api-explorer/Checkout/68/get/sessions/_sessionId_)

**Describe alternatives you've considered**
Use a manual get request outside of the library to get the status

**Additional context**

**Comment by jillingk** [comment]
Hi @majkiee,

Thanks for reaching out to us here with this feature request! This endpoint you're mentioning is indeed something we did not support yet as we we're not sure if it should be used. However after discussing internally we decided we can use this, and as a result we will add this in the next version we're releasing. The timeline for the next release is within the end of next week, hope that works for you!

A side note, I noticed you linked checkout version 68. We plan to release this only for the checkout v70 as we like to support the latest api versions in this library. Would this be any issue for you?

Best, Jilling
Adyen
